constraints/cv32e40p_core.sdc

```diff
@@ -0,0 +1,167 @@
+#
+# You may obtain a copy of the License at:
+#
+#     https://solderpad.org/licenses/SHL-2.0/
+#
+# Unless required by applicable law or agreed to in writing, software
+# and hardware implementations thereof distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESSED OR IMPLIED.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#//////////////////////////////////////////////////////////////////////////////
+# Engineer:       Arjan Bink - arjan.bink@silabs.com                         //
+#                                                                            //
+# Project Name:   CV32E40P                                                   //
+#                                                                            //
+# Description:    Example synthesis constraints.                             //
+#                                                                            //
+#                 The clock period and input/output delays are technology    //
+#                 and project dependent and are expected to be adjusted as   //
+#                 needed.                                                    //
+#                                                                            //
+#                 OBI related bus inputs arrive late on purpose and OBI      //
+#                 related outputs are available earlier (as they shall not   //
+#                 combinatorially depend o nthe OBI inputs)                  //
+#                                                                            //
+#//////////////////////////////////////////////////////////////////////////////
+
+# 200MHz
+set clock_period 5.0
+
+# Input delays for interrupts
+set in_delay_irq          [expr $clock_period * 0.50] 
+# Output delays for interrupt related signals
+set out_delay_irq         [expr $clock_period * 0.25] 
+
+# Input delays for early signals
+set in_delay_early [expr $clock_period * 0.10] 
+
+# OBI inputs delays
+set in_delay_instr_gnt    [expr $clock_period * 0.83]
+set in_delay_instr_rvalid [expr $clock_period * 0.83]
+set in_delay_instr_rdata  [expr $clock_period * 0.83]
+
+set in_delay_data_gnt     [expr $clock_period * 0.83]
+set in_delay_data_rvalid  [expr $clock_period * 0.83]
+set in_delay_data_rdata   [expr $clock_period * 0.83]
+
+# OBI outputs delays
+set out_delay_instr_req  [expr $clock_period * 0.57]
+set out_delay_instr_addr [expr $clock_period * 0.57]
+
+set out_delay_data_req   [expr $clock_period * 0.57]
+set out_delay_data_we    [expr $clock_period * 0.57]
+set out_delay_data_be    [expr $clock_period * 0.57]
+set out_delay_data_addr  [expr $clock_period * 0.57]
+set out_delay_data_wdata [expr $clock_period * 0.57]
+
+# I/O delays for non RISC-V Bus Interface ports
+set in_delay_other       [expr $clock_period * 0.10]
+set out_delay_other      [expr $clock_period * 0.57]
```

Comment:
same as before

Reply:
Hi @davideschiavone Would it be okayif I just round the numbers: 0.57 -> 0.6, 0.83 -> 0.8. The numbers will always be tech specific, which is why there is a comment at the top about that.

Reply:
Okay to just round 0.57 to 0.6?

Reply:
yes